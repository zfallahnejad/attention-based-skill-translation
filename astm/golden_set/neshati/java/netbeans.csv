992484,netbeans
2587435,netbeans
522444,netbeans
230513,netbeans
111331,netbeans
330315,netbeans
157882,netbeans
871026,netbeans
571407,netbeans
260990,netbeans
1795530,netbeans
22656,netbeans
70604,netbeans
98811,netbeans
2970947,netbeans
506855,netbeans
203907,netbeans
301607,netbeans
1880810,netbeans
2145295,netbeans
1322642,netbeans
771055,netbeans
