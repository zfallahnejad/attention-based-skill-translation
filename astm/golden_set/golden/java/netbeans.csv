992484,85,85,0.5
2587435,57,50,0.5327102803738317
522444,57,53,0.5181818181818182
230513,54,63,0.46153846153846156
111331,29,27,0.5178571428571429
330315,28,23,0.5490196078431373
157882,23,14,0.6216216216216216
871026,23,26,0.46938775510204084
571407,22,23,0.4888888888888889
260990,21,27,0.4375
1795530,21,9,0.7
22656,20,18,0.5263157894736842
70604,20,23,0.46511627906976744
98811,18,17,0.5142857142857142
2970947,13,15,0.4642857142857143
301607,12,9,0.5714285714285714
1322642,12,9,0.5714285714285714
506855,12,14,0.46153846153846156
1880810,12,6,0.6666666666666666
203907,12,15,0.4444444444444444
2145295,12,10,0.5454545454545454
2415194,10,4,0.7142857142857143
438992,10,8,0.5555555555555556
843318,10,4,0.7142857142857143
